### DISCUSSION TITLE
Add read_json_shell() function to read json data from a shell command

### DISCUSSION THREAD

**Discussion by thomas-0816** [duckdb/duckdb#6254] [Ideas]
Maybe it would be interesting to read json data produced by any kind of shell command.

```
SELECT * FROM read_json_shell_auto('<some-shell-script>');
```

Normally shell commands are run to push the data to a database, but maybe it's easier if the commands just write the json data to stdout and the output gets directly parsed.

**Comment by Alex-Monahan** [comment]
DuckDB can read from stdin if that helps! Have a look at the csv docs!
https://duckdb.org/docs/data/csv#examples

**Reply by thomas-0816** [reply]
Thanks! I have more ideas like joining shell outputs:

```
SELECT * FROM read_json_shell_auto('<some-shell-script>') a, read_json_shell_auto('<some-other-shell-script>') b where a.id = b.id;
```
